go/vt/servenv/status.go

```diff
@@ -81,6 +81,26 @@ var statusHTML = `<!DOCTYPE html>
 <html>
 <head>
 <title>Status for {{.BinaryName}}</title>
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.3/jquery.min.js"></script>
+<script>
+function refreshTableContentIfTableHasID() {
+  $.get("/debug/status", function(data) {
+    var data = $.parseHTML(data);
+    $("table").each(function() {
+  	  var findTableID = $(this).attr("id");
+  	  if (typeof findTableID !== "undefined") {
+  	    var newTable = $(data).filter("#" + findTableID);
+  	    if (newTable.length > 0) {
+  	  	  $(this).empty().append(newTable.html());
+  	    }
+  	  }
+    });
+  });
+}
+$(document).ready(function() {
+	setInterval(refreshTableContentIfTableHasID, 5000);
+});
```

Comment:
I am not sure whether we should make the interval configurable or not. Not everyone will want to overload their VTGate with a request every 5 seconds.

Reply:
totally make sense, do you mind giving me suggestions on which place is good for this config and what's the good value as a default?

Reply:
I updated this as a variable, and set it to 10 seconds as default, this should be a value that won't overload vtgate.